### PR TITLE
Image and input components

### PR DESCRIPTION
**Pull request by Manju-sree**
This PR is created for Image and Input components in CSS library.
1. Image component
-Responsive image
-Round image
2. Input component
-Input with Textbox
-Textbox with validation and errors style

### CODE REVIEW

File: Components/Input/input.css [hunk 1]
```diff
@@ -0,0 +1,74 @@
+.input-label {
+    text-align: start;
+    font-size: 1.5rem;
+    font-weight: bolder;
+    color: var(--primary-color);
+}
+
+.input-label.input-textbox {
```
Comment (JantuDeb) [inline Components/Input/input.css]: Fix the typo it's `input-label, .input-textbox{}`

**Review by JantuDeb — CHANGES_REQUESTED** [review]
Commented out some issues you need fix this.

**Review by mariyasada — APPROVED** [review]
good to merge.

File: Components/Input/input.css (hunk 1 above)
Comment (Manju-sree) [reply Components/Input/input.css]: hey! thanks for that.. i have fixed it now. please approve

**Review by JantuDeb — APPROVED** [review]
You can merge this PR as you have fix the issues.

**Pull request merged**